__tests__/page-snapshot-user-settings.tsx

```diff
@@ -0,0 +1,30 @@
+import User from 'pages/users/[id]'
+import { screen, waitForElementToBeRemoved } from '@testing-library/react'
+import { render } from 'jest.utils'
+import fetch from 'jest-fetch-mock'
+import { mockUserPage } from './page-snapshot-user'
+
+beforeEach(() => {
+  fetch.resetMocks()
+  fetch.mockResponse(mockUserPage)
+})
+
+it('renders User Settings page', async () => {
+  try {
+    const { container } = render(User, {
+      router: {
+        pathname: '/users/111/settings',
+        route: '/users/111/settings',
+        asPath: '/users/111/settings',
+        query: { id: '111' },
+      },
+    })
+    await waitForElementToBeRemoved(() =>
+      screen.queryByRole('alert', { name: 'Loading' })
+    )
+    expect(container).toMatchSnapshot()
+  } catch (e) {
+    // eslint-disable-next-line no-console
+    console.warn('cool', e)
```

Comment:
again, "cool" is not descriptive. we shouldn't be fighting with our linter. looks like you're just doing this to use Error`e` so the linter doesn't yell at you for unused variables? maybe fix that issue instead?